.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+
+# Add a reference to related issue
+Fixes: #NUMBER
+
+# Add release notes. See https://rpm-software-management.github.io/mock/Release-Notes-New-Entry
+# or add label no-release-notes if you think the change does not deserve a release notes.
```

Comment:
Contributors that need this kind of info will not have the rights to set this label :-( so I'd probably directly go to "let us know"

Reply:
updated.